$NetBSD$

Fix path of cmake_uninstall.cmake.in.
CMAKE_MODULE_PATH is changed after running of the pkgsrc wrappers.

--- CMakeLists.txt.orig	2019-03-13 20:46:48.000000000 +0000
+++ CMakeLists.txt
@@ -33,7 +33,7 @@ set(CPACK_SOURCE_IGNORE_FILES "/build;/.
 include(CPack)
 
 # Add support for the "make uninstall" target
-configure_file("${CMAKE_MODULE_PATH}/cmake_uninstall.cmake.in"
+configure_file("${CMAKE_SOURCE_DIR}/cmake/modules/cmake_uninstall.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
                IMMEDIATE @ONLY)
 add_custom_target(uninstall "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
